provider/datagen/Cargo.toml

```diff
@@ -85,6 +85,7 @@ itertools = { version = "0.10", optional = true }
 ndarray = { version = "0.15.5", default-features = false, optional = true }
 num-bigint = { version = "0.4.4", default-features = false, optional = true }
 num-rational = { version = "0.4", default-features = false, optional = true }
+num-traits = { version = "0.2.17", default-features = false}
```

Comment:
@robertbastian : if I added `optional = true`, I could not access `use num_traits::One;`. Shall I add a specific configurations?

Reply:
I don't see why you wouldn't be able to. What's the error?

Reply:
here is the error:

```
error[E0432]: unresolved import `num_traits`
  --> provider/datagen/src/transform/cldr/units/helpers.rs:12:5
   |
12 | use num_traits::One;
   |     ^^^^^^^^^^ use of undeclared crate or module `num_traits`

error[E0599]: no function or associated item named `one` found for struct `IcuRatio` in the current scope
   --> provider/datagen/src/transform/cldr/units/helpers.rs:353:29
    |
353 |         .try_fold(IcuRatio::one(), |result, &num| {
    |                             ^^^ function or associated item not found in `IcuRatio`
```

Reply:
have you added it to the experimental feature?

Reply:
yes, it seems I need to added to `experimental_components`